src/components/users/SignUpOptions.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import { Link } from 'react-router-dom';
+
+import './sign-up-options.scss';
+
+const SignUpOptions = () => {
+  return (
+    <div className="signup-options">
+      <div className="signup-options__facebook">
+        <a href="/">
```

Comment:
To my mind is high time you followed alexis's advice about move route paths as constants. I have seen href="/" many times so far, in case you wanted to change some path later on you will have to change all the occurrences along the code.